### DISCUSSION TITLE
Show indicator for new bricks on translated documents with inheritance

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#9750] [Ideas]
Hi,

new bricks added to the source document won't show up in the translated document (with inheritance).

It would be nice to add any kind of information on the translated document to display the user that there has been added a new brick to the source document and clone it directly into the translated document - without manually adding the same brick.

![image](https://user-images.githubusercontent.com/90033/125462019-7a636443-c7d4-41b7-a2f0-95a0cb359da2.png)

Any opinions about this "feature"?

Cheers,
Valentin

**Comment by dvesh3** [comment]
It shows up in my case. I added a new brick on source and it shows up on target with option to override. please see
![image](https://user-images.githubusercontent.com/5137917/125746542-44251f1a-4102-4a93-8aa2-8459333bebc0.png)

please make sure source document is specified as master in settings:
![image](https://user-images.githubusercontent.com/5137917/125746615-1a235a61-6942-40ad-b900-00414f2ce4b5.png)

**Reply by valentin-** [reply]
I've tried it again and it's working as expected.
